aten/src/ATen/native_parse.py

```diff
@@ -85,6 +85,8 @@ def type_argument_translations(arg):
     # Enables str by translating to legacy std::string.
     elif t == 'str':
         t = 'std::string'
+    elif t == 'str?':
+        t = 'std::string?'
```

Comment:
I think you'll need to add the inverse conversion [here](https://github.com/pytorch/pytorch/blob/master/tools/jit/gen_unboxing_wrappers.py#L43-L77).

Reply:
Thanks, added.